x-pack/plugins/transform/public/app/sections/transform_management/components/action_start/start_action_modal.tsx

```diff
@@ -39,6 +43,19 @@ export const StartActionModal: FC<StartAction> = ({ closeModal, items, startAndC
       buttonColor="primary"
     >
       <p>
+        {hasManagedTransforms ? (
+          <>
+            <FormattedMessage
+              id="xpack.transform.transformList.startManagedTransformsDescription"
+              defaultMessage="{transformsCount, plural, one {This transform was} other {At least one of these transforms was}} deployed as part of a module; starting {transformsCount, plural, one {it} other {them}} might impact other parts of the product."
```

Comment:
As above, the term 'module' should not be used here.

Reply:
Updated here https://github.com/elastic/kibana/pull/122305/commits/427be5b2ff13c2453e9b03611547f978baa19cbc